src/init/darwin-addusers.sh

```diff
@@ -19,14 +19,13 @@ function check_errm
 }
 
 USER=$1
-USER_MAIL=$2
-USER_REM=$3
-GROUP=$4
-INSTYPE=$5
+GROUP=$2
+DIR=$3
+INSTYPE=$4
```

Comment:
`$INSTYPE` is not used in this script as well.

Reply:
Changed it in eb724fd2d59a1c863fe35189dc67f8db42e0af14